builder/builder-git.c

```diff
@@ -51,18 +51,35 @@ git (GFile   *dir,
 
 static GFile *
 git_get_mirror_dir (const char     *url_or_path,
+                    gboolean       *is_local,
```

Comment:
It would be nice to be able to pass `NULL` to this function when one is not interested in whether the directory is local or not.